charts/hivemq-platform/values.yaml

```diff
@@ -119,7 +137,8 @@ license:
   extensions: {}
     ## Defines the name of the HiveMQ Enterprise Extension license in the Kubernetes Secret (the .elic suffix will be added automatically).
     #extension-license-name:
-    #  # Inlines the HiveMQ Enterprise Extension license as an encoded 64-bytes string.
+    #  # Inlines the HiveMQ Enterprise Extension license as a string.
+    #  # By default, this data must be a Base64 encoded string. Otherwise, set isLicenseBase64Encoded=false to use a clear string.
```

Comment:
By default, this data is expected as Base64 encoded string. Set "isLicenseBase64Encoded=false" to provide a plain string.